package.json

```diff
@@ -515,9 +515,15 @@
         "go.buildTags": {
           "type": "string",
           "default": "",
-          "description": "The Go build tags to use for all commands that support a `-tags '...'` argument",
+          "description": "The Go build tags to use for all commands, excluding `go test` (see testTags), that support a `-tags '...'` argument",
```

Comment:
`buildtags` will still be used for compiling the test files which will be using `go test`
So we should probably change this to `excluding when tests are run` or something to that effect